bayesian/api_v1.py

```diff
@@ -416,6 +417,47 @@ def post():
         return {'status': 'success'}
 
 
+class UserIntent(ResourceWithSchema):
+    method_decorators = [login_required]
+
+    @staticmethod
+    def get(user, ecosystem):
+        if user is not None and ecosystem is not None:
+            raise HTTPError(404, "Invalid user '{u}' and ecosystem '{e}'.".format(u=user,
+                                                                                  e=ecosystem))
+
+        s3 = StoragePool.get_connected_storage('S3ManualTagging')
+        # get user data
+        return s3.fetch_user_data(user, ecosystem)
+
+    @staticmethod
+    def post():
+        input_json = request.get_json()
+
+        if not input_json:
+            raise HTTPError(400, error="Expected JSON request")
+
+        if 'manual_tagging' not in input_json:
```

Comment:
This looks like using this endpoint for two different purposes. Do we really want to change endpoint behaviour based on JSON that is sent? This might be confusing.

Reply:
yes @fridex this is the requirement. where the same end point will be used for manual tagging and also for the user intent